actionpack/lib/action_dispatch/http/permissions_policy.rb

```diff
@@ -169,5 +199,90 @@ def resolve_source(source, context)
           raise RuntimeError, "Unexpected permissions policy source: #{source.inspect}"
         end
       end
+
+      def apply_mappings(sources)
+        sources.map do |source|
+          case source
+          when Symbol
+            apply_mapping(source)
+          when String, Proc
+            source
+          else
+            raise ArgumentError, "Invalid HTTP permissions policy source: #{source.inspect}"
+          end
+        end
+      end
+
+      def apply_mapping(source)
+        MAPPINGS.fetch(source) do
+          raise ArgumentError, "Unknown HTTP permissions policy source mapping: #{source.inspect}"
```

Comment:
Could you add a test for this case? I see one for `invalid` but not `unknown`

Reply:
Of course, I've added one now.